frontend/components/views/legal/TermsOfService.vue

```diff
@@ -225,9 +225,15 @@
           be made after two years from when the aggrieved party knew or should have
           known of the controversy, claim, dispute or breach.
         </p>
-
+        <p>
+          Prevailing party means the party in whose favor final judgment after appeal (if any) is rendered with
```

Comment:
The default in the American legal system is that each party has to pay their attorney's fees. This makes it easy for a well-resourced individual or group to threaten suit without merit and force capitulation as the other side runs out of money, regardless of whether they're in the right.

These changes make it to where whoever is found to be in the wrong by the courts will pay the fees of the other party's attorneys. This gives a lower-resourced party more leverage in the event they're being picked on.

Disclaimer: I am not a lawyer, this is my layman's understanding, and is not legal advice for anyone's particular situation.